json-spec/examples/landsat8-sample.json

```diff
@@ -63,6 +61,7 @@
     { "rel":"self", "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00.json"},
     { "rel":"alternate", "href": "https://landsatonaws.com/L8/153/025/LC81530252014153LGN00", "type": "html"},
     { "rel":"catalog", "href":  "http://landsat-pds.s3.amazonaws.com/L8/catalog.json"}
+    { "rel":"collection", "href": "http://landsat-pds.s3.amazonaws.com/L8/L1T-collection.json"}
```

Comment:
Should this link to the actual file in our repo? Or since this is an example we want to make it look real?

Reply:
I went back and forth on that myself. I ended up on 'make it look real', and then added a note in the readme that the extensions folder has a full example...